corehq/messaging/scheduling/forms.py

```diff
@@ -513,9 +520,9 @@ def get_layout_fields(self):
                         data_bind='with: message',
                     ),
                     data_bind=(
-                        "visible: $root.content() === '%s' || $root.content() === '%s' "
```

Comment:
A f string would make this a whole lot better.

Reply:
good call. I can make that change